packages/react-core/index.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
```

Comment:
Reason for this is mentioned here: https://github.com/patternfly/patternfly-react/pull/6791#discussion_r787988093